docs/en_US/Installation.md

```diff
@@ -1,8 +1,8 @@
 # Installation of NNI
 
-Currently we only support installation on Linux & Mac.
+Currently we support installation on Linux Mac & Windows.
 
```

Comment:
It is recommended to add a navigation bar has "linux", "mac", "windows" and "system requirements".